### ISSUE TITLE
Question about magnitude in FLOWDCN

### ISSUE THREAD

**Issue by K1NSA** [Algolzw/BSRT#14]
Dear author, I see you set the value of offset in 10*tanh(offset), what the difference between original offset and [-10,10] offset?

It seems that the magnitude is a super parameter, did you try other value? e.g. 15,20

**Comment by Algolzw** [comment]
Hi, the scale is just an empirical value, and you can change it according to your input image sizes.

**Comment by K1NSA** [comment]
Thanks a lot, and I train my work with/without pretrained Spynet weight. But weirdly, the un pretrained is better than the pretrained. Does this mean my data is not suitable with pretrained model?

**Comment by Algolzw** [comment]
Yes! It happens in many RAW image datasets because SpyNet is only trained on RGB images.

**Comment by K1NSA** [comment]
> 

Thanks for your answering, I still wonder, as training size [256,256] and test size[1050,1900].  Does the tanh(offset)*10 works for both of them?

**Comment by Algolzw** [comment]
It depends on your frames' shifts (large or small). In my experience, you could fine-tune **only the DCN part** to fit the real shifts using a large patch size (such as 512 or 768).
